src/components/Game/Game.jsx

```diff
@@ -98,49 +98,57 @@ export default function Game({exitGame, user, displayScore, questions}) {
     
   return (
     <>
-    <div className="hang-git-container">
+    <div class="page-container">
```

Comment:
Can you change all references to 'class' to 'className'.

Reply:
will do right now

Reply:
> Can you change all references to 'class' to 'className'.

done